try/run_central_instances.py

```diff
@@ -0,0 +1,2341 @@
+        self._default_namespace = server_dict.get('default_namespace', None)
+        self.enabled = True
+
+    def _required_attr(self, server_dict, attr_name, nickname):
+        try:
+            return server_dict[attr_name]
+        except KeyError:
+            raise ServerDefinitionFileError(
+                "Required server attribute is missing in definition of server "
+                "{0}: {1}".format(nickname, attr_name))
+
+    def __repr__(self):
+        return "ServerDefinition(" \
+            "nickname={s.nickname!r}, " \
+            "description={s.description!r}, " \
+            "url={s.url!r}, " \
+            "user={s.user!r}, " \
+            "password=..., " \
+            "cert_file={s.cert_file!r}, " \
+            "key_file={s.key_file!r}, " \
+            "ca_certs={s.ca_certs!r}, " \
+            "no_verification={s.no_verification!r} " \
+            "default_namespace={s.default_namespace!r})". \
+            format(s=self)
+
+    @property
+    def nickname(self):
+        """
+        Nickname of the WBEM server.
+        """
+        return self._nickname
+
+    @property
+    def description(self):
+        """
+        Short description of the WBEM server.
+        """
+        return self._description
+
+    @property
+    def url(self):
+        """
+        URL of the WBEM server.
+
+        For details see url parameter of pywbem.WBEMConnections().
+        """
+        return self._url
+
+    @property
+    def user(self):
+        """
+        User for logging on to the WBEM server.
+        """
+        return self._user
+
+    @property
+    def password(self):
+        """
+        Password of that user.
+        """
+        return self._password
+
+    @property
+    def cert_file(self):
+        """
+        Path name of file containing X.509 client certificate to be presented
+        to server, or `None` for not presenting a client certificate to the
+        server.
+        """
+        return self._cert_file
+
+    @property
+    def key_file(self):
+        """
+        Path name of file containing X.509 private key of the client
+        certificate, or `None` for not presenting a client certificate to the
+        server.
+        """
+        return self._key_file
+
+    @property
+    def ca_certs(self):
+        """
+        Path name of CA certificate file or path name of directory containing
+        CA certificate files to be used for verifying the returned server
+        certificate, or `None` for using the pywbem default locations.
+        """
+        return self._ca_certs
+
+    @property
+    def no_verification(self):
+        """
+        Boolean controlling whether the returned server certificate is to be
+        verified.
+        """
+        return self._no_verification
+
+    @property
+    def default_namespace(self):
+        """
+        Boolean controlling whether the returned server certificate is to be
+        verified.
```

Comment:
The description does not fit the attribute name.

Reply:
Fixed.  however, In the lastest code I changed to use the same class in the test code and added this property to that class.